va/caa.go

```diff
@@ -218,7 +146,7 @@ func (va *ValidationAuthorityImpl) getCAASet(ctx context.Context, hostname strin
 	// the RPC call.
 	//
 	// We depend on our resolver to snap CNAME and DNAME records.
-	results := va.parallelCAALookup(ctx, hostname, lookuper)
+	results := va.parallelCAALookup(ctx, hostname, va.dnsClient.LookupCAA)
```

Comment:
Since we no longer have multiple lookupers do we still need to pass in `va.dnsClient.LookupCAA`?

Reply:
The lookuper param was actually not touched in the legacy CAA changes, so I left it in. But looking again, it was added in 2016 for the CAA Distributed Resolver, which has since been removed. Will remove this param.